client/lib/post-normalizer/rule-create-better-excerpt.js

```diff
@@ -55,7 +55,8 @@ export function formatExcerpt( content ) {
 		}
 	} );
 
-	const betterExcerpt = trim( dom.innerHTML );
+	// trim and replace &nbsp; entities
+	const betterExcerpt = trim( dom.innerHTML.replace( /&nbsp;/g, '' ) );
```

Comment:
@roundhill did you mean to replace `&nbsp;` with `' '` instead of `''`? It looks from this diff like we're eliminating the whitespace instead of replacing it.